sci-astronomy/stellarium/stellarium-0.20.3.ebuild

```diff
@@ -17,15 +25,20 @@ SRC_URI="
 		https://github.com/Stellarium/stellarium-data/releases/download/stars-2.0/stars_8_2v0_1.cat
 	)"
 
-LICENSE="GPL-2+"
+LICENSE="GPL-2+ SGI-B-2.0"
 SLOT="0"
-KEYWORDS="amd64 ppc ppc64 ~x86 ~amd64-linux ~x86-linux"
-IUSE="debug gps media nls stars test"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug deep-sky doc gps media nls stars telescope test"
 
+# Python interpreter is used while building RemoteControl plugin
 BDEPEND="
+	${PYTHON_DEPS}
+	doc? ( app-doc/doxygen[dot] )
 	nls? ( dev-qt/linguist-tools:5 )
 "
 RDEPEND="
+	dev-libs/qtcompress:=
+	=dev-libs/qxlsx-1.3.56-r0:=
```

Comment:
Is this very restrictive dep due to your patch? Since
`=dev-libs/qxlsx-1.3*:=`
or
`=dev-libs/qxlsx-1.3.56*:=`
should also work.

Reply:
Yes. Depending on how upstream accepts my patch, I may need to change how stellarium finds it.